modules/tinymce/src/core/test/ts/webdriver/delete/ShortcutDeleteTest.ts

```diff
@@ -0,0 +1,116 @@
+import { ApproxStructure, RealKeys } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { PlatformDetection } from '@ephox/sand';
+import { TinyAssertions, TinyHooks, TinySelections } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import * as InlineFormatDelete from 'tinymce/core/delete/InlineFormatDelete';
+
+type BackspaceDeleteModifier = { ctrl: true } | { alt: true } | { meta: true };
+
+describe('webdriver.tinymce.core.delete.ShortcutDeleteTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    base_url: '/project/tinymce/js/tinymce',
+    indent: false
+  }, [], true);
+  const platform = PlatformDetection.detect();
+  const os = platform.os;
+  const browser = platform.browser;
+
+  const doBackspaceDelete = async (editor: Editor, deletionKey: 'Backspace' | 'Delete', modifiers: BackspaceDeleteModifier): Promise<void> => {
+    await RealKeys.pSendKeysOn('iframe => body', [ RealKeys.combo(modifiers, deletionKey) ]);
+    if (browser.isSafari()) {
+      // safari doesn't trigger this override automatically in test
```

Comment:
Should we then skip the test for safari since it's not really testing the right thing?

Reply:
I agree, I've skipped the entire test for Safari.